### PR TITLE
Switch to Oj

### PR DESCRIPTION
**Pull request by mezis**
Because it's faster, and it's what we're using elsewhere.

### CODE REVIEW

File: spec/controllers/topics_spec.rb [hunk 1]
```diff
@@ -40,7 +40,7 @@
       let(:data) {{
         type: 'create',
         url:  'https://example.com/widgets/123',
-        timestamp: Time.now.to_f
+        timestamp: Time.now.to_i * 1e3
```
Comment (tompave) [inline spec/controllers/topics_spec.rb:43]: Looks like you're switching from seconds to milliseconds, but you're also dropping precision. Is it required by Oj?

Comment (tompave-roo) [reply spec/controllers/topics_spec.rb:43]: ops, wrong account. it's me.

Comment (mezis) [reply spec/controllers/topics_spec.rb:43]: All timestamps in Routemaster are integer miliseconds - in this test it happened to be a float (for no reason), without failing the test.
Oj is a bit more picky so it exposed the issue with the test.

Comment (tompave-roo) [reply spec/controllers/topics_spec.rb:43]: I see, thanks.

**Comment by tompave-roo** [comment]
LGTM 👍

**Pull request merged**